lib/Dialect/EmitC/IR/EmitCDialect.cpp

```diff
@@ -50,6 +50,24 @@ void emitc::buildTerminatedBody(OpBuilder &builder, Location loc) {
   builder.create<emitc::YieldOp>(loc);
 }
 
+//===----------------------------------------------------------------------===//
+// ApplyOp
+//===----------------------------------------------------------------------===//
+
+static LogicalResult verify(mlir::emitc::ApplyOp op) {
+  // Callee must not be empty.
```

Comment:
Change `callee` to `applicableOperator` or `operator`.

Reply:
Good catch!